lib/chef-acceptance/test_suite.rb

```diff
@@ -49,10 +57,27 @@ def run
 
       exist!
 
+      FileUtils.rmtree temp_dir
```

Comment:
`run` method now is pretty complex. I would break it down into sub-soutines as below to make the testing and the readability of the code easier: 

```
def run
  create_required_files
  construct_run_command
  execute_run_command
end
```